src/app/shared/enum/history.log.ts

```diff
@@ -6,6 +6,8 @@ export enum HistoryLogTypes {
 }
 
 export enum FilterOptions {
+  dateFrom = 'dateFrom',
```

Comment:
It is better to keep the same PaskalCase for all options.
value can be in CamelCase